### DISCUSSION TITLE
Stremerbot.io Kick plugin help

### DISCUSSION THREAD

**Discussion by codycloverXO** [Sehelitar/Kick.bot#13] [General]
Hi i am newly setting up stremerbot & ur kick plugin. I got stermerbot 0.20 aswell. When I add the dlls and hit OK  I am not getting any type of authentication / kick login popup.

**Comment by Sehelitar** [comment]
Hi 👋
Did you have anything in the compilation log that could indicate that an error occured?
You might also want to check in Streamer.bot's logs files to see if there is any Kick's error logged in here.

<img width="514" alt="image" src="https://github.com/Sehelitar/Kick.bot/assets/1716704/fbe9cab0-0e6c-4214-8da8-73e47b53c378">

**Reply by codycloverXO** [reply]
Nope I am getting the compiled succesfully! As well.
[log_20231014.log](https://github.com/Sehelitar/Kick.bot/files/12907400/log_20231014.log)

**Reply by codycloverXO** [reply]
I have NordVPN running as well should I turn it off.

**Reply by Sehelitar** [reply]
The first thing to do would be to upgrade your installation to version 0.2.1, since 0.2.0 is obsolete and unsupported.
Also, which operating system do you use? (name & version)

**Reply by codycloverXO** [reply]
Sounds good I just updated & then re added everything no login popup again. 
![Screenshot 2023-10-14 204819](https://github.com/Sehelitar/Kick.bot/assets/147939712/54d6af78-1f65-4d9d-9e72-022ce1cf9002)

**Reply by Sehelitar** [reply]
If you click on the "Chat" button, does a window open?

![image](https://github.com/Sehelitar/Kick.bot/assets/1716704/db3c23f3-1d29-4841-ae94-27d6bb4ec12c)

**Reply by codycloverXO** [reply]
![Screenshot 2023-10-15 015050](https://github.com/Sehelitar/Kick.bot/assets/147939712/512c1744-0080-4559-8232-fcedf078cd29)

**Reply by Sehelitar** [reply]
Oh wow, that's what I thought. So there are 2 possibilities :
* Your Streamer.bot installation is corrupted. Try to download it from here https://streamer.bot/api/releases/streamer.bot/latest/download and copy the ``data`` folder from the current one to the new one (to keep your configuration)
* Start the bot and check if the chat feature is still broken. If it still crash, your operating system might be corrupted. You should have Microsoft Edge browser installed on your PC, is it working?

**Reply by Sehelitar** [reply]
@codycloverXO Did you have time to give this a try and check if everything is ok?
